### ISSUE TITLE
ARD/MDR Sender Link führt ins Leere

### ISSUE THREAD

**Issue by Zitronella** [mediathekview/mediathekviewweb#204]
Die Verlinkung zur ARD/MDR Seite ist defekt.
Beispiel
https://mediathekviewweb.de/ aufrufen
Erster Link zu  "Sport im Osten "SpiO"-Talk mit Marcus Burghardt"auf  "ARD" klicken ruft folgenden Link auf
https://www.ardmediathek.de/ard/player/Y3JpZDovL21kci5kZS9iZWl0cmFnL2Ntcy9mZjMzYzMxMC0wMjczLTQzMDktODllZi03MTI0OTFjZmE3ZTM dieser führt ins Leere.

Der korrekte Link wäre https://www.ardmediathek.de/video/sport-im-osten/spio-talk-mit-marcus-burghardt/mdr-de/Y3JpZDovL21kci5kZS9iZWl0cmFnL2Ntcy9mZjMzYzMxMC0wMjczLTQzMDktODllZi03MTI0OTFjZmE3ZTM/ 

bzw. würde auch reichen 

https://www.ardmediathek.de/video/Y3JpZDovL21kci5kZS9iZWl0cmFnL2Ntcy9mZjMzYzMxMC0wMjczLTQzMDktODllZi03MTI0OTFjZmE3ZTM

dann müsste nur "**ard/player**" durch "**video**" ausgetauscht werden. 
![Bild_2021-03-29---10-03-14](https://user-images.githubusercontent.com/14326582/112810604-7b5cf980-907b-11eb-8d11-c96693be2a23.png)

**Comment by bagbag** [comment]
Das ist ein Thema für den Crawler von MV. Aber bei einer Stichprobe gerade hat es funktioniert. Ist dir das aktuell noch bei Beiträgen aufgefallen?

Wenn ein Beitrag von den Mediatheken depubliziert wird, das Video aber noch erreichbar ist, dann bleibt der Eintrag in MVW, aber der link zur Mediathek geht logischerweise nicht mehr.

**Comment by Zitronella** [comment]
https://mediathekviewweb.de/#query=SpiO-Talk%20mit%20Biathlon-Bundestrainer%20Mark%20Kirchner
ARD Link ist da korrekt (https://www.ardmediathek.de/video/Y3JpZDovL21kci5kZS9iZWl0cmFnL2Ntcy9iOTA3MGU2MS0wMDkxLTQzYTctYTUwMy0xYTdjYTA1OWRkODc )
MDR Link allerdings wieder gleiches Problem. Er führt ins Leere (https://www.ardmediathek.de/ard/player/Y3JpZDovL21kci5kZS9iZWl0cmFnL2Ntcy9iOTA3MGU2MS0wMDkxLTQzYTctYTUwMy0xYTdjYTA1OWRkODc )
![Bild_2021-10-31---23-10-58](https://user-images.githubusercontent.com/14326582/139603858-c5516706-4eb6-4b9d-94d6-2b22596c612e.png)

**Comment by bagbag** [comment]
Also wenn ich da drauf klicke, werde ich zu <https://www.ardmediathek.de/video/Y3JpZDovL21kci5kZS9iZWl0cmFnL2Ntcy9iOTA3MGU2MS0wMDkxLTQzYTctYTUwMy0xYTdjYTA1OWRkODc/> weitergeleitet. Funktioniert also. Oder geht es dir darum, dass nicht direkt der finale Link da ist?

**Comment by Zitronella** [comment]
Ja, es wird weiter geleitet (manchmal), allerdings nicht unter bestimmten Umständen, nämlich nicht im privaten Modus oder wenn man Cookies und Website Daten beim Beenden von Firefox löschen lässt. 
Das habe ich jetzt endlich heraus gefunden. Allerdings ist das nun wirklich kein Problem von mediathekviewweb.de sondern liegt an der Seite ardmediathek.de und ist daher hier auch zu vernachlässigen.

Sorry, dass ich nicht schon früher darauf kam.

Nur der Vollständigkeit halber, falls von Interesse: Ich habe dazu einen Forenpost geschrieben https://www.camp-firefox.de/forum/thema/133639-website-wei%C3%9F-ob-ich-cookies-und-website-daten-beim-beenden-von-firefox-l%C3%B6schen-l/
